ironfish-rust-nodejs/src/structs/note_encrypted.rs

```diff
@@ -28,12 +28,6 @@ pub const ENCRYPTED_NOTE_PLAINTEXT_LENGTH: u32 = ENCRYPTED_NOTE_SIZE as u32 + MA
 #[napi]
 pub const ENCRYPTED_NOTE_LENGTH: u32 =
     NOTE_ENCRYPTION_KEY_LENGTH + ENCRYPTED_NOTE_PLAINTEXT_LENGTH + 96;
-//  32 value commitment
```

Comment:
is this comment calculation wrong?

Reply:
Yea it was, it also will go stale with any changes to the code above, you can hover the constant and get the value